### PR TITLE
Updating CI env with 2022.04 jcoin.datacommons.io 1649774923

### PR DESCRIPTION
**Pull request by PlanXCyborg**
Applying version 2022.04 to jcoin.datacommons.io

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# jcoin.datacommons.io
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [project_id] in explorerConfig.filters.tabs.fields not found in etlMapping
- Field error: Field [project_id] in explorerConfig.table.fields not found in etlMapping
- Field error: Field [project_id] in explorerConfig.table.fields not found in etlMapping

**Comment by PlanXCyborg** [comment]
# jcoin.datacommons.io
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [project_id] in explorerConfig.filters.tabs.fields not found in etlMapping
- Field error: Field [project_id] in explorerConfig.table.fields not found in etlMapping
- Field error: Field [project_id] in explorerConfig.table.fields not found in etlMapping

**Comment by PlanXCyborg** [comment]
# jcoin.datacommons.io
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [project_id] in explorerConfig.filters.tabs.fields not found in etlMapping
- Field error: Field [project_id] in explorerConfig.table.fields not found in etlMapping
- Field error: Field [project_id] in explorerConfig.table.fields not found in etlMapping

**Comment by PlanXCyborg** [comment]
# jcoin.datacommons.io/manifest.json
## Deployment changes
- wts
  - This version of WTS requires a DB migration (in cloud-automation deployments, the DB should be migrated automatically) (uc-cdis/workspace-token-service#40)

File: jcoin.datacommons.io/manifest.json [hunk 1]
```diff
@@ -4,28 +4,28 @@
...
     "fluentd": "fluent/fluentd-kubernetes-daemonset:v1.2-debian-cloudwatch",
-    "spark": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/gen3-spark:2022.03",
-    "hatchery": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/hatchery:2022.03",
-    "wts": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/workspace-token-service:2022.03",
-    "manifestservice": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/manifestservice:2022.03",
+    "spark": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/gen3-spark:2022.04",
+    "hatchery": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/hatchery:2022.04",
+    "wts": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/workspace-token-service:2022.04",
+    "manifestservice": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/manifestservice:2022.04",
     "ambassador": "quay.io/datawire/ambassador:1.4.2",
```
Comment (paulineribeyre) [inline jcoin.datacommons.io/manifest.json:25]: ECR?

Comment (paulineribeyre) [reply jcoin.datacommons.io/manifest.json:25]: do we even need ambassador 🤷‍♀️

Comment (atharvar28) [reply jcoin.datacommons.io/manifest.json:25]: I am not exactly familiar with ambassador service

Comment (paulineribeyre) [reply jcoin.datacommons.io/manifest.json:25]: yeah let's leave it in

**Review by paulineribeyre — APPROVED** [review]

**Pull request merged**